src/ustar_header_writer.e

```diff
@@ -214,13 +214,19 @@ feature {NONE} -- Utilities
 					{TAR_CONST}.tar_header_chksum_length)
 		end
 
-	unify_path (a_path: PATH): STRING_8
+feature -- Path helpers		
+
+	unify_utf_8_path (a_path: PATH): STRING_8
```

Comment:
renamed to show this is utf_8 ... and use PATH.components instead of guessing the separator location.